core/src/main/scripts/all-breast-su2c.sh

```diff
@@ -29,7 +29,7 @@
 ./importProfileData.pl --data $GDAC_CGDS_STAGING_HOME/brca_tcga/data_methylation.txt --meta $GDAC_CGDS_STAGING_HOME/brca_tcga/meta_methylation.txt --dbmsAction clobber
 
 # RPPA
-./importProteinArrayData.pl $GDAC_CGDS_STAGING_HOME/brca_tcga/data_rppa.txt brca_tcga
+#./importProteinArrayData.pl $GDAC_CGDS_STAGING_HOME/brca_tcga/data_rppa.txt brca_tcga
```

Comment:
maybe remove both these lines completely - no need to keep a commented out reference to a file that no longer exists. We can always look at the git history if we need to see this line